src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/MigrateVisitService.kt

```diff
@@ -158,7 +157,7 @@ class MigrateVisitService(
   }
 
   private fun isVisitTooFarInTheFuture(visitDate: LocalDateTime): Boolean {
-    return ChronoUnit.MONTHS.between(LocalDate.now(), visitDate) > migrateMaxMonthsInFuture
+    return (visitDate.toLocalDate() > LocalDate.now().plusMonths(migrateMaxMonthsInFuture))
```

Comment:
changed this as it does not allow migrating visits that are exactly n months away.